src/common/transformations/src/transformations/common_optimizations/lin_op_sequence_fusion.cpp

```diff
@@ -58,11 +58,10 @@ ov::pass::AddMultiplyFusion::AddMultiplyFusion() {
         auto new_mul = register_new_node<ov::op::v1::Multiply>(input, mul_const);
 
         // Add two constants using opset3::Add constant folding and create new Add operation
-        auto new_add =
-            std::make_shared<ov::op::v1::Add>(new_mul,
-                                              op::util::eltwise_fold<ov::op::v1::Multiply>(add_const, mul_const));
+        auto new_const = op::util::make_try_fold<ov::op::v1::Multiply>(add_const, mul_const);
```

Comment:
are there any other usages of `eltwise_fold`?
can we delete this function if no usages found

Reply:
There is one in GPU plugin src/plugins/intel_gpu/src/plugin/transformations/convert_fc_to_compressed.cpp

Reply:
can we get rid of this usage and delete `eltwise_fold` function?